### PR TITLE
Simulation enhancements

### PR DESCRIPTION
**Pull request by zah**

### CODE REVIEW

File: tests/simulation/start.sh [hunk 1]
```diff
@@ -28,8 +28,22 @@ DEFS+="-d:SECONDS_PER_SLOT=${SECONDS_PER_SLOT:-6} "  # Spec default: 12
 LAST_VALIDATOR_NUM=$(( NUM_VALIDATORS - 1 ))
 LAST_VALIDATOR="$VALIDATORS_DIR/v$(printf '%07d' $LAST_VALIDATOR_NUM).deposit.json"
 
-echo "Building $BEACON_NODE_BIN ($DEFS)"
-nim c -o:"$BEACON_NODE_BIN" $NIMFLAGS $DEFS beacon_chain/beacon_node
+build_beacon_node () {
+  OUTPUT_BIN=$1; shift
+  PARAMS="$NIMFLAGS $DEFS $*"
```
Comment (stefantalpalaru) [inline tests/simulation/start.sh]: `"$@"` is better than `"$*"`, because it surrounds each argument with double quotes, when it expands.

**Comment by arnetheduck** [comment]
how about we drop `multitail` then? they kind of do the same thing for the sim, so no point in keeping both..

**Comment by stefantalpalaru** [comment]
> how about we drop multitail then? they kind of do the same thing

What's the story on node instance termination? You exit multitail with 'q' and all the nodes are reliably killed. What are you supposed to do in tmux to get that?

**Comment by zah** [comment]
In tmux, you have `kill-session`. It's a bit more complex to execute - you need to press the tmux leader key `CTRL+B`, followed by `:kill-ses [tab][enter]`. If you want to do this more easily, you can bind a keyboard shortcut by following the instructions here:

https://superuser.com/questions/918610/creating-a-kill-current-session-shortcut-in-tmux

**Pull request merged**